main.py

```diff
@@ -756,6 +759,13 @@ def parse_args(self) -> None:
             nargs="*",
             required=False,
         )
+        parser.add_argument(
```

Comment:
we should document this argument on the readme.md

Reply:
I think Josh wanted to leave it undocumented for now. I'll double check that at stand-up as well.